polarization/tissue.py

```diff
@@ -0,0 +1,151 @@
+from .tissueStack import *
+from .pulse import *
+from copy import deepcopy
+from typing import Union
+import numpy as np
+
+__all__ = ['Tissue', 'RandomTissue2D']
+
+
+class Tissue:
+    def __init__(self, referenceStack=None, height=3000, width=200, depth=1):
```

Comment:
An initial TissueStack from which I can generate a continuous sample with varying layer thicknesses. I moved it out to RandomTissue since it's only used for generating random tissues.